webapp/src/pages/OneTimePassword.js

```diff
@@ -56,12 +57,35 @@ const OneTimePassword = () => {
       })
       .catch((err) => {
         setOtp(new Array(6).fill(""));
+        setMessage(null);
         setError(extractErrorCode(err));
         const firstOtpField = document.getElementById("otpContainer").firstChild;
         firstOtpField.focus();
       });
   };
 
+  const handleResend = (e) => {
+    e.preventDefault();
+    const firstOtpField = document.getElementById("otpContainer").firstChild;
+    api
+      .authStart({
+        phone: phoneNumber,
+        timezone: "America/New_York",
+      })
+      .then(() => {
+        setError(null);
```

Comment:
Instead of doing this all in a success handler, let's be optimistic and do this before making the API request. Then if the request errors, we can show them the error (we can also remove the duplicated code from that codepath).